unittests/test_json_serialization.py

```diff
@@ -279,12 +279,14 @@ def test_package_key_condition_expression_mapping_serialization(
                     hint_keys=["501", "502", "503"],
                     format_constraint_keys=["901", "902"],
                     requirement_constraint_keys=["1", "2", "3", "4"],
+                    repeatability_constraint_keys=["2001, 2002, 2003"],
                     package_keys=["17P"],
                 ),
                 {
                     "hint_keys": ["501", "502", "503"],
                     "format_constraint_keys": ["901", "902"],
                     "requirement_constraint_keys": ["1", "2", "3", "4"],
+                    "repeatability_constraint_keys": ["2001, 2002, 2003"],
```

Comment:
@schachmett sicher an der stelle?
```suggestion
                    repeatability_constraint_keys=["2001, 2002, 2003"],
                    package_keys=["17P"],
                ),
                {
                    "hint_keys": ["501", "502", "503"],
                    "format_constraint_keys": ["901", "902"],
                    "requirement_constraint_keys": ["1", "2", "3", "4"],
                    "repeatability_constraint_keys": ["2001", "2002", "2003"],
```

Reply:
Danke für den Hinweis, das war quatsch